src/python/pants/engine/process.py

```diff
@@ -383,17 +386,20 @@ class BinaryPathRequest:
 
     search_path: SearchPath
     binary_name: str
+    check_file_entries: bool
```

Comment:
This argument needs to be passed somewhere in the Python backend, probably?

Reply:
Yup. Working on that + test.

Reply:
Done.